### PR TITLE
STY: flake8 -- utils and tests

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Removes the flake8 exceptions for 
- utils._core
- utils.time
- tests.test_utils

## Type of change

- Style fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

via pytest --flake8

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

File: pysat/tests/test_utils.py [hunk 1]
```diff
@@ -84,24 +84,18 @@ def test_set_data_dir_bad_directory(self):
...
-            root = os.path.join(os.getenv('HOME'), '.pysat')
-            new_root = os.path.join(os.getenv('HOME'), '.saved_pysat')
-            shutil.move(root, new_root)
-            saved = True
-        except:
-            pass
+        root = os.path.join(os.getenv('HOME'), '.pysat')
+        new_root = os.path.join(os.getenv('HOME'), '.saved_pysat')
+        shutil.move(root, new_root)
+        saved = True
```
[the remark this answers is not in the document]
Comment (aburrell) [reply pysat/tests/test_utils.py:90]: Yeah...

File: pysat/tests/test_utils.py [hunk 2]
```diff
@@ -84,24 +84,18 @@ def test_set_data_dir_bad_directory(self):
...
-                # remove directory, trying to be careful
-                os.remove(os.path.join(root, 'data_path.txt'))
-                os.rmdir(root)
-                shutil.move(new_root, root)
-        except:
-            pass
+        if saved:
+            # remove directory, trying to be careful
+            os.remove(os.path.join(root, 'data_path.txt'))
+            os.rmdir(root)
```
[the remark this answers is not in the document]
Comment (aburrell) [reply pysat/tests/test_utils.py:97]: Possible dangerous answer from https://stackoverflow.com/questions/13118029/deleting-folders-in-python-recursively

```
import shutil
shutil.rmtree('/path/to/your/dir/')
```

[the remark this answers is not in the document]
Comment (jklenzing) [reply pysat/tests/test_utils.py:97]: Given the potential danger of this test, I wonder if we should have a flag so this only runs on travis

[the remark this answers is not in the document]
Comment (aburrell) [reply pysat/tests/test_utils.py:97]: I would say provide a flag that can be switched (not travis specific), and just flip it on travis.

**Pull request merged**